db/migrate/20150728014635_create_pieces.rb

```diff
@@ -8,6 +8,7 @@ def change
     	t.integer :player_id
 
     	t.timestamps
+        t.references :pieces, index: true
```

Comment:
I was thinking we'd want to have it accessible quickly. Or maybe I put the indexing in the wrong migration...you suggested putting indexing somewhere. But now that I think about it we don't want this indexing in the create_pieces. Oops.